templates/page.values.php

```diff
@@ -50,5 +50,8 @@
 		</ul>
 
 		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<div style="text-align: center;">
+			<h3><?php p($l->t('This Nextcloud is on version %s', [ \OC_Util::getVersionString() ])); ?></h3>
```

Comment:
As the rest of the steps use the theme name a lot I would suggest something more like :
`$theme->getName()` is using Nextcloud version %s

Reply:
Hm... I would actually rather do that in a follow up to this PR...
@juliushaertl is that fine?

Reply:
If we do show the version here, it should be rather short and inconspicuous. :) And it’s not a h3, a `<p>` with `color: var(--text-maxcontrast);` would be better. And shorter wording, simply `Nextcloud 22.0.0`.

Reply:
done
This is how it looks now:
![image](https://user-images.githubusercontent.com/42591237/126195511-20ca8702-16d9-400d-abed-d531179db144.png)